packages/cozy-harvest-lib/src/components/AccountForm/AccountField.jsx

```diff
@@ -47,6 +47,7 @@ export class AccountField extends PureComponent {
     const {
       disabled,
       hasError,
+      hasInitialValues,
```

Comment:
I find this prop name a little bit confusing, especially with initialValues right below. It does not describe enought what it does. But a better name is not easy to find. What to you think about `mode="edition"` or maybe better `forceEncryptedPlaceholder` ? After all, it is exactly what we want to do here.

Reply:
I also prefer `forceEncryptedPlaceholder` 👍